dora/src/bytecode/writer.rs

```diff
@@ -888,6 +968,24 @@ impl BytecodeWriter {
         self.emit_values(&values);
     }
 
+    fn emit_store_field(
```

Comment:
I think this is the same as `emit_load_field`, I think we could rename this function to e.g. `emit_field` and then use it for both stores and loads.

Reply:
I will remove the duplicate, but maybe we should be a little bit more specific for the name? What do you think about `emit_field_access` (or `emit_access_field`)?

Reply:
Ok